tiled/server/schemas.py

```diff
@@ -8,6 +8,8 @@
 import pydantic.generics
 
 from ..structures.core import StructureFamily
+from .pydantic_array import ArrayStructure
+from .pydantic_dataframe import DataFrameStructure
```

Comment:
This can be changed to import from `tiled.structures.dataframe` once #229 is merged.